src/components/Movie/index.tsx

```diff
@@ -0,0 +1,33 @@
+import Link from "next/link";
+import Image from "next/image";
+import style from './style.module.scss';
+import { MovieType } from "@/util/types";
+
+//TODO: Change src on line :21 to correct property for images.
+
+/*
+*  Renders a link for from given props that has a background of a
+*  movie poster. The poster can either be an image served statically
+*  or a link to an external source. Each instance of this component is 
+*  clickable and routes to /movies/:id 
+*/
+export default function Movie({movieData}: {movieData: MovieType}) {
+    const link = `/movies/${movieData.id}`
+  return (
+    <>
+      <Link
+        style={{ textDecoration: "none" }}
+        href={link}
+        className={style.card}
+      >
+        <Image
+          className={style.img}
+          height={120}
+          width={90}
+          src={"/dummy.jpg"}
+          alt={`The poster for ${movieData.title}`}
+        />
+      </Link>
```

Comment:
Möjligen att man skulle kunna lägga in detta i en figure tag för att bli av med att man kan navigera sig vidare genom att klicka i gapet som finns mellan bilderna. Men är en bagatell i sammanhanget.
Vet som sagt inte vad som är semantiskt korrekt :)

Reply:
Tack för feedback, Ska experimentera lite med styling och researcha lite mer om det.